src/components/Profile/OnchainIdentityForm.js

```diff
@@ -126,12 +126,13 @@ function OnchainIdentityForm({
       )}
       {defaultValues.hasUserWarn && !isUserWarnAccepted && (
         <Paragraph>
-          <Alert
-            type="warning"
-            message={(
+          <Result
```

Comment:
why call this result ?

Reply:
That is the name of the ANTD component https://ant.design/components/result/